src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1040,9 +1040,14 @@ public static double Min(double val1, double val2)
             // otherwise returns the larger of the inputs. It
             // treats +0 as larger than -0 as per the specification.
 
-            if (val1 != val2 && !double.IsNaN(val1))
+            if (val1 != val2)
```

Comment:
Pre-existing, but the above comment looks wrong: "otherwise returns the larger of the inputs"... but this is min, not max.

Reply:
Fixed and changed it to use greater vs lesser rather than larger vs smaller, as the latter two terms are potentially confusing.